test/Microsoft.AspNetCore.Authentication.Test/Google/GoogleMiddlewareTests.cs

```diff
@@ -781,6 +789,13 @@ private static TestServer CreateServer(GoogleOptions options, Func<HttpContext,
                         {
                             await context.Authentication.ChallengeAsync("Google");
                         }
+                        else if (req.Path == new PathString("/tokens"))
+                        {
+                            var authContext = new AuthenticateContext(TestExtensions.CookieAuthenticationScheme);
```

Comment:
The gun is for me, this syntax is literally killing me (though it's absolutely not specific to this PR). 

> Le 17 févr. 2016 à 21:44, Hao Kung <notifications@github.com> a écrit :
> 
> In test/Microsoft.AspNetCore.Authentication.Test/Google/GoogleMiddlewareTests.cs:
> 
> > @@ -781,6 +789,13 @@ private static TestServer CreateServer(GoogleOptions options, Func<HttpContext,
> >                          {
> >                              await context.Authentication.ChallengeAsync("Google");
> >                          }
> > +                        else if (req.Path == new PathString("/tokens"))
> > +                        {
> > +                            var authContext = new AuthenticateContext(TestExtensions.CookieAuthenticationScheme);
> What's the gun for? I can't nuke the scheme because its a required argument, I could switch it to automatic scheme but that's just less clear I think
> 
> —
> Reply to this email directly or view it on GitHub.
>